examples/basic/cli/src/examples/fetchEmployeePageByAdUsername.ts

```diff
@@ -27,6 +27,9 @@ export async function fetchEmployeePageByAdUsername(
   const result = await client.objects.Employee.where({
     $and: [{ adUsername }, { employeeNumber: { ne: 5 } }],
   }).fetchPageOrThrow();
+  // for await (const e of client.objects.Employee.asyncIter()) {
+  //   e.__name;
+  // }
```

Comment:
Reference for future.

Reply:
Yeah I like the object set being iterable itself (though that might be a bit harder to discover)

Reply:
Original demo had that. I took it off for now because we wanted to push people into downselecting and the where syntax was not established. This felt easier to support for now while we figure out what we want